scripts/ci/update_rerun_py_and_c_version.py

```diff
@@ -33,10 +33,13 @@ def update_python_line(line: str, version_line: str, version_info_line: str) ->
 
 def set_rerun_py_version(init_path: Path, version: semver.VersionInfo) -> None:
     version_line = f'__version__ = "{version}"\n'
-    version_info_line = (
-        # TODO(andreas): We released `0.18.0` `"None"` as prerelease, but really it should be `None`.
-        f'__version_info__ = ({version.major}, {version.minor}, {version.patch}, "{version.prerelease}")\n'
-    )
+    version_info_items = list(str(item) for item in (version.major, version.minor, version.patch))
+    if version.prerelease is not None:
+        version_info_items.append(f'"{version.prerelease}"')
+    else:
+        version_info_items.append("None")
```

Comment:
This looks like a bug

```suggestion
        version_info_items.append(None)
```

Reply:
It's not a bug, the output is the same regardless if `"None"` or `None` is used.

One source of inspiration would be [`sys.version_info`](https://docs.python.org/3/library/sys.html#sys.version_info), but in fairness we are not adhering exactly to that standard. I'm unsure we should care that much though. Most package have a `__version__` string, but rarely have `__version_info__`.